config/routes.rb

```diff
@@ -1,3 +1,15 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'events#index'
+
+  get       '/signup',  to: 'users#new'
+  post      '/signup',  to: 'users#create'
+  get       '/login',   to: 'sessions#new'
+  post      '/login',   to: 'sessions#create'
+  delete    '/logout',  to: 'sessions#destroy'
+  resources 'users',    only: [:new, :create, :show]
+  resources 'sessions', only: [:new]
+  resources 'events',   only: [:index, :new, :create] do
+    post :checkin
```

Comment:
since both routes belong to Attendance model, they should be built within attendance controller.

Reply:
moved the  methods.